### PR TITLE
update text

### LINKED ISSUE #1060: [OTHER] Update KendallDoesCoding.json
### What would you like to share?

Update my bio

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by KendallDoesCoding**
## Fixes Issue
- Fixes #1060 

## Changes proposed
- Update text on my bio

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers
I forgot I copied my bio from my website, I needed to frame it further.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1058"></a>

### CODE REVIEW

File: public/data/KendallDoesCoding.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Kendall Pinto",
   "type": "personal",
-  "bio": "Hello. My name is Kendall. I am a Software Developer / Coder. I have made a bunch of programs in various languages, like Python, Javascript, and the one I'm using to make this website, HTML & CSS.",
+  "bio": "Hello. My name is Kendall. I am a Software Developer / Coder. I have made a bunch of programs in various languages, like Python, Javascript, and the one I'm current working on projects with, HTML & CSS.",
```
Comment (syedsohan) [inline public/data/KendallDoesCoding.json:4]:
```suggestion
  "bio": "Hello, My name is Kendall. I am a Software Developer / Coder. I have made a bunch of programs in various languages, like Python, JavaScript, and the one I'm currently working on projects with, HTML & CSS.",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Comment by KendallDoesCoding** [comment]
@emmalearnscode How do I put a hyperlink in my bio?

**Review by syedsohan — APPROVED** [review]

**Comment by emmalearnscode** [comment]
> @emmalearnscode How do I put a hyperlink in my bio?

You add it to the links array.

**Comment by KendallDoesCoding** [comment]
> > @emmalearnscode How do I put a hyperlink in my bio?
> 
> You add it to the links array.

I did that but I want it in the text lol

**Comment by KendallDoesCoding** [comment]
A hyperlink inside this text for example, can I use a, a href and put my hyperlink there?

image

**Review by emmalearnscode — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
> A hyperlink inside this text for example, can I use a, a href and put my hyperlink there?
> 
> image

So this not possible?

**Review by schmelto — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!, Thanks for your contribution :)

**Pull request merged**